layouts/index.html

```diff
@@ -10,11 +10,15 @@
   <body>
     <main>
       <div class="section hero">
-        <div class="hero-text">
-          <p>Serving up</p>
-          <h1>TASTY</h1>
-          <h1>BYTES</h1>
-          <p class="two">since 2014</p>
+        <div class="hero-text-background-container">
+          <div class="hero-text">
```

Comment:
I should research how to create knockout text which reveals a changing background, or...
* container with z-index 0 is black background with burger with dynamically changing text
* container on top of that is just the burger background image
* there is a layer mask added on the burger background image

OR

* Is it possible to create an image that has transparency that will reveal a container below it?

Later, of course